src/DssSpell.t.sol

```diff
@@ -751,47 +902,544 @@ contract DssSpellTest is DssSpellTestBase {
         // assertEq(gov.balanceOf(address(pauseProxy)), prevMkrPause);
     }
 
-    function testDirectCompV2Integration() public {
+    function testDirectCompV2Integration() private {
```

Comment:
should we just remove this? looks like a spell test not something directly generalizable

Reply:
rm'ed